src/containers/VisitPage.js

```diff
@@ -95,8 +95,9 @@ class VisitPage extends Component {
     getUserConfirmation.unset();
   }
 
-  loadClient = ( clientContainer ) => {
-    this.setState({ client: clientContainer.client });
+  loadClient = (clientContainer) => {
+    const client = Object.assign(cloneDeep(CLIENT_DEFAULTS), clientContainer.client);
```

Comment:
OK thanks, one more change, I believe we want to Object.assign separately on client.current and client.future, in order to merge their fields rather than overwriting them entirely.